lib/mj-single.js

```diff
@@ -315,6 +316,28 @@ function ConfigureMathJax() {
     }
   };
 
+  if (extensions) {
+    //
+    // Parse added extensions list and add to standard ones
+    //
+    var extensionList = extensions.split(/s*,\s*/);
+    for (var i = 0; i < extensionList.length; i++) {
+      var matches = extensionList[i].match(/^(?:(.+)\/)?(.*)$/);
+      var category = matches[1], name = matches[2];
+      if (!category) {
+        window.MathJax.extensions = window.MathJax.extensions.concat(name + '.js');
+      } else {
+        if (!window.MathJax[category]) {
+          window.MathJax[category] = {};
+        }
+        if (!window.MathJax[category].extensions) {
+          window.MathJax[category].extensions = [];
+        }
+        window.MathJax[category].extensions =
+          window.MathJax[category].extensions.concat(name + '.js');
+      }
+    }
+  }
```

Comment:
You might want to consider allowing `--extensions "TeX/noUndefined.js"` as well as `--extensions "TeX/noUndefined"`.  In that case, you only want to add the `.js` if it isn't already there.